concrete/src/Search/Pagination/PermissionablePagination.php

```diff
@@ -2,22 +2,21 @@
 namespace Concrete\Core\Search\Pagination;
 
 use Concrete\Core\Search\ItemList\ItemList;
+use Concrete\Core\Support\Facade\Facade;
 use Pagerfanta\Adapter\ArrayAdapter;
 use Pagerfanta\Pagerfanta;
 
 /**
- * Processes a thousand requests and builds pagination out of them.
- * This is slow on larger sites, but will yield accurate pagination even with permissions
- * Caveat: The most you can process in one result set is 1000 results. Otherwise, use PagerPagination
- * or disable permissions.
+ * Processes requests and builds pagination out of them.
  */
 class PermissionablePagination extends Pagination
 {
-    protected $maxResultsToProcessAtOnce = 1000;
```

Comment:
What about keeping this protected property (to be initialized with the value read from the configuration in the constructor)?
Removing protected properties isn't good for backward compatibility...

Reply:
Yeah you're right